static/app/views/automations/components/actionFilters/subfiltersList.tsx

```diff
@@ -0,0 +1,192 @@
+import styled from '@emotion/styled';
+import {uuid4} from '@sentry/core';
+
+import {Flex} from 'sentry/components/container/flex';
+import {Button} from 'sentry/components/core/button';
+import AutomationBuilderInputField from 'sentry/components/workflowEngine/form/automationBuilderInputField';
+import {RowLine} from 'sentry/components/workflowEngine/form/automationBuilderRowLine';
+import AutomationBuilderSelectField from 'sentry/components/workflowEngine/form/automationBuilderSelectField';
+import {PurpleTextButton} from 'sentry/components/workflowEngine/ui/purpleTextButton';
+import {IconAdd, IconDelete} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {MatchType} from 'sentry/views/automations/components/actionFilters/constants';
+import {useDataConditionNodeContext} from 'sentry/views/automations/components/dataConditionNodes';
+
+export function SubfiltersList() {
+  const {condition, condition_id, onUpdate} = useDataConditionNodeContext();
+
+  const subfilters = condition.comparison.filters || [];
+  const subfilterCount = subfilters.length;
+
+  function addSubfilter() {
+    const newSubfilters = [
+      ...subfilters,
+      {
+        id: uuid4(),
+        match: MatchType.EQUAL,
+      },
+    ];
+    onUpdate({filters: newSubfilters});
+  }
+
+  function removeSubfilter(id: string) {
+    const newSubfilters = subfilters.filter(
+      (subfilter: Record<string, any>) => subfilter.id !== id
+    );
+    onUpdate({filters: newSubfilters});
+  }
+
+  function updateSubfilter(id: string, comparison: Record<string, any>) {
+    const newSubfilters = subfilters.map((subfilter: Record<string, any>) => {
+      if (subfilter.id === id) {
+        return {
+          ...subfilter,
+          ...comparison,
+        };
+      }
+      return subfilter;
+    });
+    onUpdate({filters: newSubfilters});
+  }
+
+  return (
+    <Flex column>
+      <Flex column>
+        {subfilters.map((subfilter: Record<string, any>, i: number) => {
+          return (
+            <SubfilterRow
+              subfilter={subfilter}
+              subfilter_id={`${condition_id}.comparison.filters.${subfilter.id}`}
+              onRemove={() => removeSubfilter(subfilter.id)}
+              onUpdate={comparison => updateSubfilter(subfilter.id, comparison)}
+              key={subfilter.id}
+              isLastRow={i === subfilterCount - 1}
+            />
+          );
+        })}
+      </Flex>
+      <PurpleTextButton
+        borderless
+        icon={<IconAdd />}
+        size="xs"
+        onClick={() => addSubfilter()}
+      >
+        {t('Sub-filter')}
+      </PurpleTextButton>
```

Comment:
the lack of padding on this button looks weird in hover state
![image](https://github.com/user-attachments/assets/b5237c78-d40a-4aef-8b2a-9705db7000b2)

Reply:
fixed

<img width="193" alt="Screenshot 2025-05-20 at 3 07 10 PM" src="https://github.com/user-attachments/assets/c6aabec1-4923-4a2d-8003-23787a890e02" />